develop/interoperability/xcm-config.md

```diff
@@ -0,0 +1,245 @@
+---
+title: XCM Config
+description: Learn how the XCM Executor configuration works for your custom Polkadot SDK-based runtime with detailed guidance and references.
+---
+
+# XCM Config
+
+## Introduction
+
+The [XCM executor](https://github.com/paritytech/polkadot-sdk/tree/master/polkadot/xcm/xcm-executor/src){target=\_blank} is a crucial component responsible for interpreting and executing XCM messages with Polkadot SDK-based chains. It processes and manages XCM instructions, ensuring they are executed correctly and in sequentially. Adhering to the Cross-Consensus Virtual Machine (XCVM) specification, the XCM executor can be customized or replaced with an alternative that also complies with the [XCVM standards](https://github.com/polkadot-fellows/xcm-format?tab=readme-ov-file#12-the-xcvm){target=\_blank}.
```

Comment:
Where are we with "XCM messages"? We usually call them just XCMs, for Cross-Consensus Messages, instead of Cross-Consensus Messaging messages. Is this something we've talked about before in the docs?

Reply:
I like XCMs, but I think we need to clarify it somewhere. So, probably since this is the first appearance of it, we can do something like "XCM messages (XCMs)," and then directly use XCMs